graphql_query_examples/graphql_query_examples.md

```diff
@@ -915,17 +915,28 @@ query filters{
         }
         htmlUrl
       }
+      pageInfo{
+        endCursor
+        hasNextPage
+      }
     }
   }
 }
 
-query services_by_filter{
-  account{
-    services(filterIdentifier:{id: "Z2lkOi8vb3BzbGV2ZWwvRmlsdGVyLzg1NA"}){
-      nodes{
+query services_by_filter($endCursor: String) {
+  account {
+    services(
+      filterIdentifier: {id: "Z2lkOi8vb3BzbGV2ZWwvRmlsdGVyLzIwNjg"}
```

Comment:
Long term we probably shouldn't use real ID's in these examples.  I would replace them with `XXXX` or something "fake" just for safety.